manifests/odh/params.env

```
@@ -0,0 +1,5 @@
+# Injected variables from the Operator
+odh-dashboard-image=quay.io/opendatahub/odh-dashboard:main
+admin_groups=odh-admins
+dashboard-url=https://www.redhat.com
+section-title=Open Data Hub
```

```suggestion
section-title=OpenShift Open Data Hub
```
should we just have this set to the correct value by default? same as for the other 2 params.env files. 
Operator can still inject this value

---

Not sure it matters -- I was just trying to put _some value_ there -- I could make it more of a description than an actual value. I do not think it's important to be aligned on this. It was more of an example than anything else.

---

ok, i thought you want to put the "real" value here. but if it is just placeholder, i am fine with any value